packages/terra-application-links/src/tabs/ApplicationTabs.scss

```diff
@@ -0,0 +1,146 @@
+:local {
+  .application-tabs {
+    display: flex;
+    flex: 1 1 auto;
+    flex-wrap: nowrap;
+    justify-content: center;
+    padding-left: var(--terra-application-tabs-spacer-width, 2rem);
+    padding-right: var(--terra-application-tabs-spacer-width, 2rem);
+    position: relative;
+    width: 100%;
+  }
+
+  .tabs-container {
+    display: flex;
+    flex: 1 1 auto;
+    flex-wrap: nowrap;
+    justify-content: center;
+    overflow: hidden;
+    position: relative;
+    width: calc(100% - 2 * var(--terra-application-tabs-spacer-width, 2rem));
+
+    &:focus {
+      outline: none;
+    }
+  }
+
+  .tabs-container.is-calculating {
+    visibility: hidden;
+
+    /*
+     * During calculations tabs should display with auto size and not grow or shrink in order to get
+     * content width for trunction logic
+     */
+    .tab {
+      flex: 0 0 auto;
+    }
+  }
+
+  .tab,
+  .tab-menu span {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    word-break: normal;
+    word-wrap: normal;
+  }
+
+  .tab {
+    flex: 0 1 auto;
+    max-width: var(--terra-application-tab-max-width, 16.714rem);
+    min-width: var(--terra-application-tab-min-width, 8.357rem);
+    text-align: center;
+
+    .label:nth-child(2) {
+      margin-left: var(--terra-application-tabs-margin-next-to-icon, 0.357rem);
+    }
+  }
+
+  .tab-menu {
+    align-items: baseline;
+    display: flex;
+    flex: 1 0 auto;
+    justify-content: center;
+    max-width: 100%;
+    min-width: var(--terra-application-tab-menu-min-width, 7.143rem);
+
+    span {
+      flex: 0 1 auto;
+      min-width: 0;
+      white-space: normal;
+    }
+
+    svg {
+      flex: 0 0 auto;
+      font-size: 0.571rem;
+      margin-left: var(--terra-application-tabs-menu-arrow-left-margin, 0.5rem);
+    }
+  }
+
+  .tab,
+  .tab-menu {
+    background-image: var(--terra-application-tabs-background-image, linear-gradient(to bottom, #007ee0, #007ee0));
+    background-position: var(--terra-application-tabs-background-position, top-left);
+    background-repeat: no-repeat;
+    background-size: var(--terra-application-tabs-background-size, 0 4px);
+    border-color: var(--terra-application-tabs-border-color, #dedfe0);
+    border-style: solid;
+    border-width: var(--terra-application-tabs-border-width, 0 1px 0 0);
```

Comment:
I am assuming the border width is the +2 in the calculation above. Should we allow this to be themeable?

Reply:
Changed the logic to use getClientBoundingReact().width.